db/seeds.rb

```diff
@@ -1,153 +1,153 @@
 
 CategoryItem.create(item_id: 2, category_id: 1)
 
 Item.create(title: "Bacon, Egg, and Cheddar Scones",
-		description: "Fresh cheddar cheese mixed with bacon and egg",
-		price: 1500,
-		image: open(path + "scones.jpg"))
+    description: "Fresh cheddar cheese mixed with bacon and egg",
+    price: 1500,
+    image: open(path + "scones.jpg"))
 
 CategoryItem.create(item_id: 3, category_id: 1)
 
 #brunch items --------------------------------------------------------------
 Item.create(title: "Casserole",
-		description: "Delicious casserole filled with bacon",
-		price: 1200,
-		image: open(path + "casserole.jpg"))
+    description: "Delicious casserole filled with bacon",
+    price: 1200,
+    image: open(path + "casserole.jpg"))
 
 CategoryItem.create(item_id: 4, category_id: 2)
 
 Item.create(title: "Asparagus Mushroom Bacon Quiche",
-		description: "Nothing better than a quiche with bacon",
-		price: 1100,
-		image: open(path + "quiche.jpg"))
+    description: "Nothing better than a quiche with bacon",
+    price: 1100,
+    image: open(path + "quiche.jpg"))
 
 CategoryItem.create(item_id: 5, category_id: 2)
 
 Item.create(title: "Grilled Cheese French Toast With Bacon",
-		description: "Grilled cheese.  French toast.  Bacon.  Nuff said",
-		price: 1500,
-		image: open(path + "grilled_cheese.jpg"))
+    description: "Grilled cheese.  French toast.  Bacon.  Nuff said",
+    price: 1500,
+    image: open(path + "grilled_cheese.jpg"))
 
 CategoryItem.create(item_id: 6, category_id: 2)
 
 #Lunch ----------------------------------------------------------------
 
 Item.create(title: "Bacon-Wrapped Jalapeno Poppers",
-		description: "Super spicy!",
-		price: 1000,
-		image: open(path + "pepper.jpg"))
+    description: "Super spicy!",
+    price: 1000,
+    image: open(path + "pepper.jpg"))
 
 CategoryItem.create(item_id: 7, category_id: 3)
 
 Item.create(title: "Bacon Wrapped Chicken",
-		description: "Farm raised chicken wrapped in bacon, and grilled to perfection",
-		price: 1500,
-		image: open(path + "bacon_wrapped_chicken.jpg"))
+    description: "Farm raised chicken wrapped in bacon, and grilled to perfection",
+    price: 1500,
+    image: open(path + "bacon_wrapped_chicken.jpg"))
 
 CategoryItem.create(item_id: 8, category_id: 3)
 
 Item.create(title: "Bacon Potato Soup",
-		description: "Hot soup with a touch of bacon",
-		price: 800,
-		image: open(path + "soup.jpg"))
+    description: "Hot soup with a touch of bacon",
+    price: 800,
+    image: open(path + "soup.jpg"))
 
 CategoryItem.create(item_id: 9, category_id: 3)
 
 # Dinner -------------------------------------------------------------
 Item.create(title: "Sweet Bacon-Wrapped Venison Tenderloin",
-		description: "Freshly hunted venison wrapped in crispy bacon.  It's a meal that is deer to me.",
-		price: 2600,
-		image: open(path + "venison.jpg"))
+    description: "Freshly hunted venison wrapped in crispy bacon.  It's a meal that is deer to me.",
```

Comment:
Line is too long. [100/80]<br>Align the elements of a hash literal if they span more than one line.